doc/releases/luminous.rst

```diff
@@ -8,7 +8,7 @@ luminous squid (watasenia scintillans, aka firefly squid).
 v12.2.13 Luminous
 =================
 
-This is the 13th bug fix release of the Luminous v12.2.x long term stable
+This is the 13th bug fix release of the Luminous v12.2.x long-term stable
```

Comment:
can you revert those? I don't think you're improving things here.

Reply:
Sure, done.